datasets/emotone_ar/emotone_ar.py

```diff
@@ -0,0 +1,83 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Dataset of 10065 tweets in Arabic for Emotion detection in Arabic text """
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+
+import datasets
+
+
+_CITATION = """\
+@inbook{inbook,
+author = {Al-Khatib, Amr and El-Beltagy, Samhaa},
+year = {2018},
+month = {01},
+pages = {105-114},
+title = {Emotional Tone Detection in Arabic Tweets: 18th International Conference, CICLing 2017, Budapest, Hungary, April 17–23, 2017, Revised Selected Papers, Part II},
+isbn = {978-3-319-77115-1},
+doi = {10.1007/978-3-319-77116-8_8}
+}
+"""
+
+
+_DESCRIPTION = """\
+Dataset of 10065 tweets in Arabic for Emotion detection in Arabic text"""
+
+
+_HOMEPAGE = "https://github.com/AmrMehasseb/Emotional-Tone"
+
+
+_DOWNLOAD_URL = "https://raw.githubusercontent.com/AmrMehasseb/Emotional-Tone/master/Emotional-Tone-Dataset.csv"
+
+
+class EmoToneAr(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class EmotoneAr(datasets.GeneratorBasedBuilder):
```

This must be the camel case version of the dataset script name `emotone_ar`